extensions/dataseeding/catalog/src/main/java/org/eclipse/dataspaceconnector/dataseeding/catalog/CatalogDataseedingExtension.java

```diff
@@ -237,9 +187,11 @@ private void saveAssets(String connectorId) {
             assetIndexLoader.accept(asset3, dataAddress3);
             assetIndexLoader.accept(asset4, dataAddress4);
             assetIndexLoader.accept(asset5, dataAddress5);
+            return List.of(asset1, asset2, asset3, asset4, asset5);
         } catch (EdcException ex) {
             ex.printStackTrace();
         }
+        return Collections.emptyList();
```

Comment:
In case of an issue when seeding the Assets, I would prefer to throw an exception, preventing the connector to boot instead of having the connector in an undesirable state.